runtime/lua/vim/lsp/protocol.lua

```diff
@@ -296,6 +296,12 @@ local constants = {
     -- Base kind for an organize imports source action
     SourceOrganizeImports = 'source.organizeImports';
   };
+
+  Encodings = {
+    UTF8 = 'utf-8',
+    UTF16 = 'utf-16',
+    UTF32 = 'utf-32'
+  }
 }
```

Comment:
So far this table is reserved for things that are defined in the specification. Afaik `Encodings` is not part of it?

Reply:
Yeah it isn't, but I think that `offsetEncoding` isn't an official part of it yet either? Do you know if there is a better place to put these constants?